string name
float64 price
int64 auction_ID
